doc/nrf/releases_and_maturity/software_maturity.rst

```diff
@@ -1189,7 +1189,7 @@ The following table indicates the software maturity levels of the support for Go
         - Supported
         - Experimental
```

Comment:
Please check if we can already apply similar change to the nRF5340 SoC.

Reply:
Discussed offline - it is not possible at the moment